worker/src/__tests__/eval-service.test.ts

```diff
@@ -391,9 +391,6 @@ describe("create eval jobs", () => {
     expect(jobs.length).toBe(1);
     expect(jobs[0].project_id).toBe("7a88fb47-b4e2-43b8-a06c-a5ce950dc53a");
     expect(jobs[0].job_input_trace_id).toBe(traceId);
-    console.log(jobs[0]);
-    const j = await getEvalQueue()?.getJob(jobs[0].id);
-    console.log(j);
```

Comment:
Aside from logging the result `j` was never asserted. Hence, I removed it. Happy to revert if it serves a purpose or adjust the assertions.